src/main/kotlin/no/nav/familie/ef/sak/behandling/migrering/MigreringService.kt

```diff
@@ -282,15 +307,17 @@ class MigreringService(
         return behandlingService.hentBehandling(behandling.id)
     }
 
-    private fun validerSimulering(behandling: Behandling) {
+    private fun validerSimulering(fagsak: Fagsak, behandling: Behandling, ignorerFeilISimulering: Boolean) {
         val simulering = simuleringService.hentLagretSimmuleringsresultat(behandling.id)
         val oppsummering = simulering.oppsummering
+        val inneholderEtterbetaling = oppsummering.etterbetaling.compareTo(BigDecimal.ZERO) != 0
+
         if (oppsummering.feilutbetaling.compareTo(BigDecimal.ZERO) != 0) {
             throw MigreringException(
                 "Feilutbetaling er ${oppsummering.feilutbetaling}",
                 MigreringExceptionType.SIMULERING_FEILUTBETALING
             )
-        } else if (oppsummering.etterbetaling.compareTo(BigDecimal.ZERO) != 0) {
+        } else if (inneholderEtterbetaling && !ignorerFeilISimulering) {
```

Comment:
Kan det noen gang bli tilbakekreving, og ikke etterbetaling? 🤔

Reply:
Ja, men det skal nog stoppe på `oppsummering.feilutbetaling.compareTo(BigDecimal.ZERO)` då
Lurte først på om vi skulle åpne opp for det og, men tenker vi venter litt med det :)

Reply:
👍